### DISCUSSION TITLE
Sign the software

### DISCUSSION THREAD

**Discussion by toinouH** [cafali/SnapKey#48] [New Features & Ideas]
Hello, I would like to request to ship the software with certificates.

It would make the software looks cleaner in regards to the end user and to the OS.

You can get certificates for free as your software is open source according to multiple google search results like:
https://about.signpath.io/product/open-source or https://www.certum.eu/en/code-signing-certificates/

Thank you :)

Keep up, great software ^^

**Comment by cafali** [comment]
Hi,

Thank you for your kind words and for the suggestion. I appreciate the links you provided and will look into getting certificates/code-signing for SnapKey.

As this is my first time handling this, I’ll need to do some research to ensure everything is set up correctly.

Thanks again for your input and support!
